src/main/java/Duke.java

```diff
@@ -1,10 +1,71 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+    private static void listToDo(List<Task> tdList){
+        System.out.println("Here are the tasks in your list:");
+        int idx = 1;
+        for(Task todo: tdList){
+            System.out.println(String.format("%d. %s",idx,todo.getStatus()));
+            idx += 1;
+        }
+    }
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello, I'm \n" + logo);
+        System.out.println("What can I do for you?\n");
+
+        List<Task> toDoList = new ArrayList<Task>();
+
+        Scanner in = new Scanner(System.in);
+        while(true){
+            String input = in.nextLine();
+            String[] cmd = input.split(" ");
+            String main_cmd = cmd[0];
+            if (cmd.length == 0){
+                System.out.println("Please type something\n");
+                continue;
+            }
+            if (main_cmd.equals("bye")){
+                break;
+            } else if (main_cmd.equals("list")){
+                listToDo(toDoList);
+            } else if (main_cmd.equals("mark")) {
+                System.out.println("Nice! I've marked this task as done: ");
+                int idx = Integer.parseInt(cmd[1]) - 1;
+                // add try except later
+                toDoList.get(idx).markDone();
+                System.out.println(toDoList.get(idx).getStatus());
+            } else if (main_cmd.equals("unmark")) {
+                System.out.println("OK, I've marked this task as not done yet:");
```

Comment:
I think that it would be better to abstract some methods out to make the code look neater, for example for unmark and mark.